airbyte-integrations/connectors/source-github/acceptance-test-config.yml

```diff
@@ -70,3 +103,38 @@ acceptance_tests:
     tests:
       - config_path: "secrets/config.json"
         configured_catalog_path: "integration_tests/configured_catalog.json"
+        ignored_fields:
```

Comment:
I think that this means we are sampling a popular repository in our tests.  I think that the tradeoff is worth it - information in more stable streams (organizations, repositories) is ignored to get us juicier problems in PRs and issues.